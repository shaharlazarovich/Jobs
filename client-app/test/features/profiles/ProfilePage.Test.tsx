import { ReactWrapper } from "enzyme";
import React from "react";
import { findByTestAttr } from '../../common/testUtils';
import ProfilePage from "../../../src/features/profiles/ProfilePage";
import { setupWrapper } from "../../common/wrapper";
import { Route } from "react-router-dom";
import { match } from '../../common/mocks';

describe('Profile Page UniTest', () => {
    let wrapper: ReactWrapper;
    beforeEach(async () => {
        wrapper = await setupWrapper(<Route component={ProfilePage} />);
    });
    test('renders without error', () => {
        const component = findByTestAttr(wrapper, 'component-profile-page');
        expect(component.length).toBe(1);
    });
    test('should match the snapshot', () => {
        const component = findByTestAttr(wrapper, 'component-profile-page');
        expect(component).toMatchSnapshot();
    });
})