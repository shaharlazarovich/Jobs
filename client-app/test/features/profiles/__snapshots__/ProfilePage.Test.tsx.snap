// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Page UniTest should match the snapshot 1`] = `null`;
